jscomp/syntax/src/res_printer.ml

```diff
@@ -427,6 +427,16 @@ let isValidNumericPolyvarNumber (x : string) =
          | _ -> false)
   else a >= 48
 
+let isTupleArray (expr : Parsetree.expression) =
```

Comment:
Can you move this to `res_parsetree_viewer`? That file contains all AST helpers.

Reply:
Updated!